tox.ini

```diff
@@ -110,7 +110,8 @@ changedir = {toxinidir}
 deps =
     {env:LSR_ANSIBLE_DEP:ansible}
     docker
-    molecule<3
+    podman
```

Comment:
Adding podman does not seem to belong in this commit. Also it seems to depend on an environment variable and does not seem to be needed all the time.

Reply:
> Adding podman does not seem to belong in this commit. Also it seems to depend on an environment variable and does not seem to be needed all the time.

We still must support the docker runtime for travis, and I suppose many developers may want to use docker by default since podman is relatively new.

How do you suggest writing this such that running in the travis CI environment, and by default, docker is used, but as a developer I can run molecule with the podman driver without having to change any files?

Reply:
Sure - installing both `docker` and `podman` isn't "clean" - but what's the harm? And does the cost of doing some really elegant solution justify the benefit?

Reply:
> Adding podman does not seem to belong in this commit

One of the big reasons for moving to molecule v3 is support for podman.

Reply:
It is also possible to install podman on the travis Ubuntu machines:
https://travis-ci.community/t/podman-libpod-support/6823/2

> > Adding podman does not seem to belong in this commit
> 
> One of the big reasons for moving to molecule v3 is support for podman.

Since you moved it into its own commit, you addressed this nonetheless.

> I suppose many developers may want to use docker by default since podman is relatively new.

This seems to be worth evaluating to ensure we do not spend unnecessary effort on supporting docker when everyone might be happ(ier) with podman.

Reply:
> This seems to be worth evaluating to ensure we do not spend unnecessary effort on supporting docker when everyone might be happ(ier) with podman.

I would rather do that in a future commit/PR.  I know (anecdotally) that there are a lot of system roles developers who still use `docker`, and I don't want to force them to move to `podman`.  All I really wanted to do in this PR was to give system roles developers the ability to run molecule with either `podman` or `docker`.  So far, the effort to support `docker` is neither unnecessary nor inordinate.

OK?